src/main/kotlin/com/icat/orboarding/user/application/ports/inbound/AuthenticationServicePort.kt

```diff
@@ -0,0 +1,10 @@
+package com.icat.orboarding.user.application.ports.inbound
+
+import com.icat.orboarding.user.adapters.inbound.dtos.request.LoginDTO
+import com.icat.orboarding.user.adapters.inbound.dtos.response.TokenDTO
+import org.springframework.security.core.userdetails.UserDetailsService
+
+interface AuthenticationServicePort : UserDetailsService {
+
+    fun authenticate(loginDTO: LoginDTO): TokenDTO
```

Comment:
```suggestion
    fun authenticate(loginDTO: LoginDTO): TokenDomain
```
Opa maninho, seguindo as regrinhas da arquitetura, aqui teria que retornar um TokenDomain, por exemplo, pois se não o application acaba tendo código da camada de adapters